framework/src/media/audio/audio_manager.c

```diff
@@ -159,18 +160,30 @@ static audio_manager_result_t set_audio_volume(audio_card_type_t type, uint8_t v
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
-static audio_manager_result_t find_audio_card_if_empty(audio_card_type_t card_type)
+static audio_manager_result_t init_audio_stream(audio_card_type_t card_type)
 {
-	int id = g_actual_audio_out_card_id;
+	if (((card_type == INPUT) && (g_actual_audio_in_card_id >= 0)) || ((card_type == OUTPUT) && (g_actual_audio_out_card_id >= 0))) {
```

Comment:
What I meaning is that 'g_actual_audio_in_card_id >= 0' means 'get_actual_audio_in_card_id()' already processed.
so just return AUDIO_MANAGER_SUCCESS is correct.
Because it is initialized once init called, and there's no logic that change it now.
(Plan is that we will provide apis to change device or change card in future)

Reply:
Okay. Modified.